torch/_inductor/ir.py

```diff
@@ -4401,6 +4401,7 @@ class ExternKernelNode:
 
 class FallbackKernel(ExternKernelAlloc):
     args_default_value: List[Dict[str, Any]]
+    args_name: List[str]
```

Comment:
Can we just extend `args_default_value` to include names instead of adding another class member?